docs/pages/blog/turbo-1-2-0.mdx

```diff
@@ -0,0 +1,108 @@
+
+- [**New Task Filtering API**](#new-task-filtering-api): `--filter` flag adds more expressive and intuitive task filtering capabilities to `turbo run`.
+- [**Human-readable and JSON dry runs**](#debug-and-automate-with---dry-run): `--dry-run` flag can print out information about a `turbo run` without executing any tasks, in both human and JSON-parse friendly formats.
+- [**Improved Internal Scheduler and Graph**](#improved-internal-scheduler-and-graph): We refactored `turbo` 's internal scheduler and graph to be more ergonomic and predictable
+- [**Enhanced Remote Cache Security**](#cache-outputs-integrity-and-signature-verification): Cryptographically sign remote cache artifacts with your own key phrase
+
+Update today by running `npm install turbo@latest`. After running `turbo run` for the first time, you'll see instructions about how to use `@turbo/codemod` to run automatic migrations for `v1.2`.
+
+## New Task Filtering API
+
+We are excited to release one of our most requested features: the ability to expressively filter tasks through a `--filter` flag. The `--filter` flag is the much more intuitive successor to the current combination of `--scope`, `--include-dependencies`, `--since`, and `--no-deps` flags. With `--filter` you can tell `turbo` to restrict executing commands to a subset of matched packages in your monorepo based on name, folder, and by whether it has changed since a git commit ref like so:
+
+- `--filter=<package_name>` - match by exact package name or glob pattern
+- `--filter=...<package_name>`- match by package name/glob and include all dependent packages of match(es)
+- `--filter=...^<package_name>`- match by package name/glob and include all dependent packages of match(es), but exclude itself/themselves
+- `--filter=<package_name>...` - match by name and include all matched package's dependencies
+- `--filter=<package_name>^...` - match by name/glob and include all matched package dependencies, but not itself/themselves
+- `--filter={./path/to/package}` - match by path or filesystem glob pattern
+- `--filter=[origin/main]` - match by changed packages since a git commit
+
+You can use multiple filters together to get even more granular filtering as well as combine each part of the above patterns `{}`, `[]` , `^` , and `...` to express more complex behavior.
+
+For example, assuming you have an app located in `./apps/web` directory, some local packages that it uses as dependencies, and a Turborepo pipeline where `test` depends on `^build` topologically, running:
+
+```jsx
+turbo run test --filter={./apps/web}[HEAD^1]^...
+```
+
+would tell `turbo` to ensure that dependencies are built and to run the `test` script in all of the local dependencies of the app located in `./apps/web`, not including that app itself, if the app has changed since HEAD^1.
+
+We believe `--filter` is more expressive and intuitive than `--scope`. In the future, we will be slowly deprecating the `--scope` flag.
```

Comment:
```diff
+ We believe `--filter` is more expressive and intuitive than `--scope`. In the future, we will be slowly deprecating the `--scope` flag.
- We believe `--filter` is more expressive and intuitive than `--scope` and plan to deprecate the `--scope` flag in a future release.
```

No such thing as slowly deprecating? I would pitch that we make sure we have a good migration story first, deprecate it, and aim for removal in 2.0?